tests/manager/scripts/test_playtime_entries.py

```diff
@@ -529,6 +529,7 @@ def playtime(
 
 
 class TestPlaytimeEntriesEmailReportsScript:
+    @freeze_time("2021-01-01")
```

Comment:
🚨 this is just a placeholder to get tests passing until https://github.com/ThePalaceProject/circulation/pull/2094 gets merged. This line should be removed before this gets merged in.

Reply:
This has been removed.